strange issue building on openSUSE Tumbleweed

### Expected behavior
Build correctly
### Actual behavior
exits with error 
`org.codehaus.mojo:buildnumber-maven-plugin:3.0.0:create (default) on project phoenicis: Execution default of goal org.codehaus.mojo:buildnumber-maven-plugin:3.0.0:create failed: Illegal pattern character 't'
`### Steps to reproduce
install build necessities on openSUSE Tumbleweed (a rolling but tested Linux OS)
`sudo zypper in maven fakeroot gradle`
use `git clone git://github.com/PhoenicisOrg/phoenicis.git `to download current (not https://)
use the current workaround to build  `mvn -DskipTests clean package`

### System information
* software version (git revision) of phoenicis and scripts repositories
* git (tool) version 2.34.1 
* ```
Apache Maven 3.8.1 (SUSE 3.8.1-3.2)
Maven home: /usr/share/maven
Java version: 17.0.1, vendor: N/A, runtime: /usr/lib64/jvm/java-17-openjdk-17
Default locale: en_GB, platform encoding: UTF-8
OS name: "linux", version: "5.15.7-1-default", arch: "amd64", family: "unix"

```
* used operating system
* openSUSE Tumbleweed
NAME="openSUSE Tumbleweed"
# VERSION="20211215"


* used Java version
```
openjdk 11.0.13 2021-10-19
OpenJDK Runtime Environment (build 11.0.13+8-suse-2.2-x8664)
OpenJDK 64-Bit Server VM (build 11.0.13+8-suse-2.2-x8664, mixed mode)
```
(note mvn shows a different java version, this one is output of java --version)

I am sorry some real life issues and inability to run virtual stuff (no vt-x) on this Mac Mini took me this long.

I am actually surprised to say that this issue happens on Ubuntu LTS with the same steps. Same strange error on console. Keep in mind that I can test things and/or provide more info (as maven says).

[INFO] Reactor Summary for phoenicis 5.0-SNAPSHOT:
[INFO] 
[INFO] phoenicis .......................................... FAILURE [01:02 min]
[INFO] phoenicis-configuration ............................ SKIPPED
[INFO] phoenicis-entities ................................. SKIPPED
[INFO] phoenicis-multithreading ........................... SKIPPED
[INFO] phoenicis-win32 .................................... SKIPPED
[INFO] phoenicis-tools .................................... SKIPPED
[INFO] phoenicis-repository ............................... SKIPPED
[INFO] phoenicis-scripts .................................. SKIPPED
[INFO] phoenicis-library .................................. SKIPPED
[INFO] phoenicis-engines .................................. SKIPPED
[INFO] phoenicis-settings ................................. SKIPPED
[INFO] phoenicis-containers ............................... SKIPPED
[INFO] phoenicis-javafx ................................... SKIPPED
[INFO] phoenicis-cli ...................................... SKIPPED
[INFO] phoenicis-dist ..................................... SKIPPED
[INFO] phoenicis-tests .................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:03 min
[INFO] Finished at: 2021-12-24T21:48:16+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:buildnumber-maven-plugin:3.0.0:create (default) on project phoenicis: Execution default of goal org.codehaus.mojo:buildnumber-maven-plugin:3.0.0:create failed: Illegal pattern character 't' -> [Help 1]                                                          
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

--- a/pom.xml
+++ b/pom.xml
@@ -56,6 +56,7 @@
         <spring.version>5.3.14</spring.version>
         <formatter-maven-plugin.version>2.17.1</formatter-maven-plugin.version>
         <graalvm.version>21.3.0</graalvm.version>
+        <buildnumber.version>3.0.0</buildnumber.version>
     </properties>
     <dependencyManagement>
         <dependencies>
@@ -120,7 +121,7 @@
                                     <pluginExecutionFilter>
                                         <groupId>org.codehaus.mojo</groupId>
                                         <artifactId>buildnumber-maven-plugin</artifactId>
-                                        <versionRange>[1.4,)</versionRange>
+                                        <version>${buildnumber.version}</version>
                                         <goals>
                                             <goal>create</goal>
                                         </goals>
@@ -169,13 +170,12 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>buildnumber-maven-plugin</artifactId>
-                <version>3.0.0</version>
+                <version>${buildnumber.version}</version>
                 <configuration>
                     <revisionOnScmFailure>unknown</revisionOnScmFailure>
                     <getRevisionOnlyOnce>true</getRevisionOnlyOnce>
                     <doCheck>false</doCheck>
                     <doUpdate>false</doUpdate>
-                    <timestampFormat>{0, date, yyyy-MM-dd HH:mm}</timestampFormat>
                 </configuration>
                 <executions>
                     <execution>

